src/blr/cli/init.py

```diff
@@ -0,0 +1,93 @@
+"""
+Create and initialize a new analysis directory.
+"""
+import logging
+import os
+import os.path
+import sys
+from pathlib import Path
+from importlib_resources import read_binary
+
+from ..utils import guess_paired_path
+
+logger = logging.getLogger(__name__)
+
+
+CONFIGURATION_FILE_NAME = "blr.yaml"
+
+
+def add_arguments(parser):
+    parser.add_argument(
+        "--reads1",
+        "--r1",
+        type=Path,
+        metavar="READS",
+        help="First paired-end read file (.fastq.gz). The second is found automatically.",
```

Comment:
```suggestion
        help="First paired-end read file (.fastq.gz). The second is found automatically.",
        required=True
```

Instead of having to check if `args.read1` is `None` in the main functions you could just set it as an required argument.

Reply:
Good point, I forgot argparse can do that.